index.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html lang="en">
+
+    <head>
+        <meta charset="UTF-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <title>Portfolio | Edwin Gichuhi</title>
+        <link rel="stylesheet" href="style.css">
+    </head>
+
+    <body>
+        <header class="header">
+            <nav class="nav">
+                <a href="#">My Logo</a>
+                <span><img src="./assets/icons/Union.png" alt="menu"></span>
+            </nav>
+        </header>
+        <section class="headline-section">
+            <p class="intro-text">I'm Edwin, Glad to see you!</p>
+            <p class="details-text">I’m a software developer! I can help you build a product, feature or website.
+                Look through some of my work and experience! If you like what you see and have
+                a project you need coded, don’t hesitate to contact me.
+            </p>
```

Comment:
- [ ] it seems that you've implemented all the styles but I see that the font-weight is not working please make sure that your header is the same as the one in the Figma design 
- [ ] Also try to remove the space between the header and paragraph it should be the same as in the Figma design 

- Yours 
<img width="284" alt="Screen Shot 2022-09-28 at 17 47 38" src="https://user-images.githubusercontent.com/61790587/192841761-a528b707-d0b9-455a-b0b9-87eb5203ff1f.png">
- Figma 
<img width="259" alt="Screen Shot 2022-09-28 at 17 39 47" src="https://user-images.githubusercontent.com/61790587/192841803-57903c0a-14e3-46d8-905e-d999b0e2775d.png">

Reply:
Fixed.